### PR TITLE
Feature/sb 4 users router

### PR DESCRIPTION
**Pull request by eccentricnode**
## Feature/Problem

User app routes

## Solution

N/A

## Areas of Impact

N/A

## UI Images

N/A

## Manual Testing
N/A
- Verify CI passes

### CODE REVIEW

File: client/apps/user-portal/src/app/router.module.ts [hunk 1]
```diff
@@ -0,0 +1,21 @@
+import { NgModule } from '@angular/core';
+import { Routes, RouterModule } from '@angular/router';
+import { SermonsComponent, SermonListComponent } from '@app/sermons';
+
+
+const routes: Routes = [
+    {
```
Comment (VicAv99) [inline client/apps/user-portal/src/app/router.module.ts]: There is a lot of spacing here.

Comment (eccentricnode) [reply client/apps/user-portal/src/app/router.module.ts]: I will work on that.

File: client/apps/user-portal/src/app/router.module.ts [hunk 2]
```diff
@@ -0,0 +1,20 @@
+import { NgModule } from '@angular/core';
+import { Routes, RouterModule } from '@angular/router';
+import { SermonsComponent, SermonListComponent } from '@app/sermons';
+
+const routes: Routes = [
+  { 
+    path: '', component: SermonsComponent, children: [
```
Comment (VicAv99) [inline client/apps/user-portal/src/app/router.module.ts]: So, I think the components are a little off.

The way the `users` app works is that the main route will have a list of sermons for people to scroll through.
Once a user selects a video, the should be routed to a view of that specific video they selected.

Comment (eccentricnode) [reply client/apps/user-portal/src/app/router.module.ts]: So the main route is to, the sermons component, but some sort of 'sermons-view' should be available on the '/:id' route. Correct?

Comment (eccentricnode) [reply client/apps/user-portal/src/app/router.module.ts]: Would the sermon-view component be usable in the admin app as well as the users?

Comment (VicAv99) [reply client/apps/user-portal/src/app/router.module.ts]: As of now, no, the admin would not see the `sermon-view` component.

**Pull request merged**